## Synthesis
- 
## Source [^1]
- A glass slide, silicon wafer, or glass bead on which are deposited biomolecules or other material in a regular micro-scale pattern to enable automated simultaneous multiple assays of target substances or activities. Microarrays are powerful analytical tools with wideranging applications, including the analysis of gene expression and of genetic variation. They can be designed to carry small DNA molecules (see DNA MICROARRAY), proteins (e.g. antibodies or antigens), carbohydrates or other organic molecules, or even individual living cells. These reagents are applied to the substrate in a regular microscopic grid pattern, each being identified by its unique coordinate, or address, on the grid. Interaction of a target substance (e.g. an antibody or a complementary nucleic acid) with a particular address on the microarray activates or attaches a label (e.g. a fluorescent dye). The microarray can then be 'read' by a scanner, which automatically assesses the amount of label at each address, and hence the amount of target substance. Even smaller-scale nanoarrays have been developed, to increase further the scope and speed of this technology. These are made using a form of high-precision lithography, and employ atomic force microscopy to detect direct interactions between individual molecules with nanometer resolution. No label is required, although some applications do use reporter labelling. Advances in next-generation DNA sequencing mean that it is replacing microarray analysis for some applications, particularly where sensitivity and accuracy are at a premium, e.g. for assessing rare or previously unknown sequence variants (see RNA SEQUENCING). See Feature.
### Microarray Technology
- A living cell makes and uses thousands of different RNAs and proteins to perform all its vital functions. Real insight into cellular metabolism requires knowledge of which components are being expressed by different cell types, for example during development, under different physiological conditions, or in disease states, and how their concentrations change over time.
- Microarrays were developed in the 1990s for the automated analysis of thousands of different biomolecules simultaneously. They have enabled researchers to study the function of genes and their products (RNAs and proteins) on a genome-wide basis$\textemdash$the field of functional genomics. These techniques generate massive amounts of data, and computerized handling, analysis, and presentation of the data are crucial for valid interpretation.
#### Examples of microarrays
- DNA microarrays ('gene chips' or 'DNA chips')
- Contain short oligonucleotides or larger DNA fragments that hybridize with complementary DNA or RNA sequences. Uses include:
	- following expression patterns of whole genomes over time;
	- identifying chemical changes throughout a cell's chromosomal DNA;
	- identifying and measuring minute deletions and duplications of DNA, e.g.in cancer cells.
- Protein microarrays
	- Contain microsamples of purified proteins.
	- Used to determine protein-protein interactions at a cellular scale.
- Tissue microarrays
	- Contain small thin sections of tissue taken from different tissue locations or pathology specimens.
	- Used to identify distribution of specific proteins throughout a tissue sample.
- Chemical microarrays
	- Contain chemical compounds.
	- Used, for example, to select potential drugs with particular chemical binding properties.
#### Preparation of DNA microarrays
- There are two basic types of DNA microarrays.
	- In situ oligonucleotide arrays (DNA chips)
		- DNA oligonucleotides (20-25 nucleotides) are assembled in situ on a quartz wafer or glass slide, using a photolithography process similar to that used to make microprocessor chips.
		- A single chip may contain up to a million specified oligonucleotides at regularly spaced sites called addresses, or 'features'.
		- Poor or incorrect hybridization leads to signal 'noise'.
	- Spotted microarrays
		- Based on the principle of the dot-blot technique with complementary DNA obtained from clones of the genes of interest.
		- Assembled by spotting minute amounts of denatured DNA onto a coated glass microscope slide, so that the DNA dries and becomes attached to the slide.
		- Contain large DNA fragments (hundreds to thousands of base pairs long).
		- Up to 20,000 features per slide.
		- Cheaper and easier to make$\textemdash$can be made 'in house' by researchers.
- ![[Screenshot 2025-09-11 at 5.07.49 PM.png]]
	- Measuring mRNAs using an oligonucleotide microarray
	- Parts
		- Nucleic acids labelled with fluorescent tag (in some cases mRNAs are made into complementary DNAs, which are tagged)
		- mRNA (or complementary dNA)
			- fluorescent label
		- Tagged nucleic acids incubated with the microarray to allow complementary sequences to hybridize and bind to each other
		- fluorescing addresses
		- oligonucleotide microarray
		- Computer
		- Intensity of fluorescence at each address quantified by a computer and values adjusted for background noise; the result corresponds to the amount of mRNA (or cDNA) at each address
		- Excess solution washed away and microarray analyzed by scanner with a fluorescence detector
		- Scanner
		- Interactions at micro-level
		- oligonucleotide
		- fluorescent tag
		- mRNA sample to be tested
## References

[^1]: [[(Home Page) A Dictionary of Biology 8th Edition by Oxford Reference]]